charts/oes/charts/spinnaker/config/orca-local-pipeline-promotion.yml

```diff
@@ -307,8 +286,10 @@ job:
                         value: "${{execution.startTime} .toString()}"
                       - name: user
                         value: "${execution.authentication.user}"
-                    image: 'opsmxdev/apline-with-curl'
+                    image: 'opsmxdev/policy:v1'
```

Comment:
Clarification: So, we put the script into the image now?

Reply:
yes Abhinay